frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -24,6 +37,8 @@ export const ScheduledEventsTable: FunctionComponent<ScheduledEventTableProps> =
           interval_type={event.interval_type}
           repeat={event.repeat}
           start_date={event.start_date}
+          onClick={handleClick(offset + index + 1)}
```

Comment:
This won't do ... Say you have scheduled events 1, 2, and 3 ... What happens when I delete scheduled event 2? 3 will move into position 2, but that ID will no longer exist. Use the individual IDs for each event, those returned by the API.